src/trade-params/TradeParamsValidation.js

```diff
@@ -1,10 +1,9 @@
-const epochWithinRange = (epoch, range) =>
-    !!(range.find(r => {
-        const millis = epoch * 1000;
-        const startMillis = r.open[0].getTime();
-        const endMillis = (r.close[1] || r.close[0]).getTime();
-        return millis >= startMillis && millis <= endMillis;
-    }));
+const epochWithinRange = (epoch, range) => {
+    const millis = epoch * 1000;
+    const startMillis = range.open[0].getTime();
+    const endMillis = (range.close[1] || range.close[0]).getTime();
+    return (millis >= startMillis && millis <= endMillis);
+};
```

Comment:
This breaks because in this case, range is not an array . `export const durationValid = (duration, durationUnit, options) =>
    !!options.find(opt => durationUnit === opt.unit && duration <= opt.max && duration >= opt.min);
`